board.py

```diff
@@ -39,3 +40,49 @@ def __setitem__(self, pos: str, item: Piece):
         row = int(pos[1]) - 1
         column = self.columns[pos[0]]
         self.matrix[row][column] = item
+
+    def inside_board(self, i: int, j: int) -> bool:
+        return 0 <= i < 8 and 0 <= j < 8
+
+    def idxs_to_coordinates(self, i: int, j: int) -> str:
+        row = str(i + 1)
+        col = self.inv_col[j]
+        return col + row
+
+    def coordinates_to_idxs(self, coord: str) -> Tuple[int, int]:
+        i, j = int(coord[1]) - 1, self.columns[coord[0]]
+        return (i, j)
+
+    def valid_moves(self, position: str) -> List[str]:
```

Comment:
Add docstrings to all of these functions in this fashion (https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html). This means for `inside_board`, `idxs_to_coordinates`,  and `coordinates_to_idxs` as well. The docstrings should make it obvious what the input and return type are (for example, it is not obvious what the input string should be for coordinate or position) as well as what the List[str] output is an output of.

For example, it is not intuitive what the `position` should be in the format "A7". The docstring should capture this